db/migrate/20210602150743_change_active_status_to_registration_status.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeActiveStatusToRegistrationStatus < ActiveRecord::Migration[6.1]
+  def up
+    rename_column :passengers, :active_status, :registration_status
+  end
+  def down
+    rename_column :passengers, :registration_status, :active_status
+  end
```

Comment:
I feel like you should embrace `change` to keep up with the times, but it doesn't really matter. What _does_ matter though is that I see that you didn't actually run this migration, or at least didn't commit your schema changes when you did!

Reply:
ahh whoops!